collectors/build/jenkins-audit/src/test/java/com/capitalone/dashboard/collector/AuditCollectorTest.java

```diff
@@ -0,0 +1,69 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.model.CollectorType;
+import com.capitalone.dashboard.repository.AuditCollectorRepository;
+import com.capitalone.dashboard.repository.AuditResultRepository;
+import com.capitalone.dashboard.repository.DashboardRepository;
+import com.capitalone.dashboard.service.DashboardAuditService;
+import org.assertj.core.api.AssertionsForInterfaceTypes;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.scheduling.TaskScheduler;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class AuditCollectorTest {
+
+    private AuditCollectorTask testee;
+    private TaskScheduler taskScheduler;
+    private DashboardRepository dashboardRepository;
+    private DashboardAuditService dashboardAuditService;
+    private AuditResultRepository auditResultRepository;
+    private AuditCollectorRepository auditCollectorRepository;
+
+
+    @Before
+    public void setup() {
+        taskScheduler = mock(TaskScheduler.class);
+        auditResultRepository = mock(AuditResultRepository.class);
+        auditCollectorRepository = mock(AuditCollectorRepository.class);
+        dashboardRepository = mock(DashboardRepository.class);
+        dashboardAuditService = mock(DashboardAuditService.class);
+        AuditCollectorSettings auditCollectorSettings = new AuditCollectorSettings();
+        auditCollectorSettings.setServers(Arrays.asList("http://localhost:8081/"));
```

Comment:
please move these to properties

Reply:
This is unit test class, we were ok with it.